policy/release/collection/rule-data-validation.rego

```diff
@@ -0,0 +1,6 @@
+#
+# METADATA
+# title: rule-data-validation
```

Comment:
Perhaps something shorter like:

```suggestion
# title: policy-data
```

Reply:
Decided to go with `policy_data` since that's the case rego usually expects.